tests/kernel/wave/wave_gemm_test.py

```diff
@@ -93,7 +94,7 @@ def repeat(acc: tkl.Register[M, N, tkl.f32]) -> tkl.Register[M, N, tkl.f32]:
             gemm(a, b, c)
             iree_ref = torch.zeros(2048, 10240, dtype=torch.float32)
             generate_iree_ref("mmt", [a, b], [iree_ref], config)
-            assert torch.equal(c, iree_ref)
+            assert_allclose(c.numpy(), iree_ref.numpy())
```

Comment:
Can you check to see if torch.assert_allclose works the same? Then we dont have to convert to numpy?

Reply:
`torch.assert_allclose` is deprecated, switched to `torch.assert_close`, error message now looks like:
```
E           AssertionError: Tensor-likes are not close!
E
E           Mismatched elements: 20971517 / 20971520 (100.0%)
E           Greatest absolute difference: 1413.0994873046875 at index (1486, 268) (up to 1e-05 allowed)
E           Greatest relative difference: 63904244.0 at index (243, 8383) (up to 1.3e-06 allowed)
```